src/app/revisors/interceptor.service.ts

```diff
@@ -0,0 +1,32 @@
+import { HttpEvent, HttpHandler, HttpInterceptor, HttpRequest, HTTP_INTERCEPTORS } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { TokenService } from '../services/token.service';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7132249)
```suggestion
import { TokenService } from "../services/token.service";
```